Demo review comments changes (#92)
etn-ccis/blui-react-native: docs/src/componentDocs/CollapsibleHeaderLayout/markdown/CollapsibleHeaderLayoutExamples.mdx

## Collapsible Header Layout

> The `CollapsibleHeaderLayout` component supports expansion and collapse events from clicks to the header section. When using as a top level layout component, it will gradually collapse and expand relative to the scroll position of the screen.

Reviewer: ```suggestion
The `CollapsibleHeaderLayout` is a full-screen layout component that automatically collapses and expands the Header component as the view is scrolled.
```
